cmd/kube-apiserver/app/server.go

```diff
@@ -465,9 +465,25 @@ func BuildGenericConfig(s *options.ServerRunOptions, proxyTransport *http.Transp
 		return nil, nil, nil, nil, nil, fmt.Errorf("failed to create admission plugin initializer: %v", err)
 	}
 
+	// Read client cert/key for plugins that need to make calls out
+	certBytes, keyBytes := []byte{}, []byte{}
+	if len(s.ProxyClientCertFile) > 0 && len(s.ProxyClientKeyFile) > 0 {
```

Comment:
Wow, this is the wrong cert/key pair.  Given the generic case of webhook admission from a generic apiserver, this key pair should be signed by the the API server's client CA.

@lavalamp I thought we talked about this before you wired it.  You need a *different* client cert/key pair.  Re-using this one leaves people making apiservers to consume the webhook in a bad spot.
@kubernetes/sig-auth-bugs